### ISSUE TITLE
Are you going to put out a new release with vulkan support?

### ISSUE THREAD

**Issue by mgateno** [skiselkov/librain#25]
I see you have made new commits since your last release? Do we have to compile our own? Are you going to put out a new release?

**Comment by dilshaan123** [comment]
Seems like a lot of us asking questions regarding Vulkan if this plugin will be compatible with Vulkan or not since some of the payware aircraft that we bought did Metin saying it is compatible with librain , it will be better of you to at least come out with bit of communication and say if it is possible or your working on it , some kind of information will be better.
Thanks 
Mate

**Comment by Brewlab** [comment]
Agree, it will be nice to know the release milestones at least... I cant see any information recently. THX

**Comment by dilshaan123** [comment]
Since it’s a free plugin he actually doesn’t have to talk much, I think best way to know the answers is to ask developers of paid aircraft which we paid them and was sold saying it is compatible with librain plugin . He can keep quite since it’s a free plugin.

**Comment by Brewlab** [comment]
@dilshaan123, For sure everyone is free to reply or not but it is more about the community, If I create something for others free or not and make it open whatever it is, as part of that community it is kind to follow-up even for few words... ( of course personal reason might be behind where I will understand....)

**Comment by dilshaan123** [comment]
Hello @ Saso Kiselkov are you doing something about librain for Vulkan ???

**Comment by dilshaan123** [comment]
Nothing on librain ???

**Comment by kukel** [comment]
Lots of commits for vulkan back in May so it might already be compatible. 
https://github.com/skiselkov/librain/commit/9c02c0b32819782ba37b7bae8459c070f86c3689

**Comment by wegylexy** [comment]
So when will be the next release?
